crates/pubnub_transport_reqwest/src/reqwest.rs

```diff
@@ -13,9 +14,12 @@ struct TransportReqwest {
 #[async_trait::async_trait]
 impl Transport for TransportReqwest {
     async fn send(&self, request: TransportRequest) -> Result<TransportResponse, PubNubError> {
+        let path = prepare_path(&request.path, &request.query_parameters);
+        info!("{} {}", request.method, path);
+        let requet_url = format!("{}{}", &self.hostname, path);
```

Comment:
```suggestion
        let request_url = format!("{}{}", &self.hostname, path);
```

Reply:
@mohitpubnub should we print `hostname` somewhere in logs?

Reply:
YESS!! That's what My suggestion was during discussion.